incubator/hnc/test/e2e/acting-as.go

```diff
@@ -0,0 +1,52 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "sigs.k8s.io/multi-tenancy/incubator/hnc/pkg/testutils"
+)
+
+var _ = Describe("Acting-as", func() {
+	const (
+		nsTarget = "target"
+		saTeamA  = "team-a"
+		saTeamB  = "team-b"
+	)
+
+	BeforeEach(func() {
+		CleanupNamespaces(nsTarget)
+	})
+
+	AfterEach(func() {
+		CleanupNamespaces(nsTarget)
+	})
+
+	It("should allow acting as different service accounts", func() {
+		MustRun("kubectl create ns", nsTarget)
+		MustRun("kubectl get ns", nsTarget)
+		MustRun("kubectl get sa -n", nsTarget)
```

Comment:
What's the purpose of these two lines? If `nsTarget` doesn't get created instantly, line 29 will just repeat until it does.